x/interchainstaking/keeper/callbacks.go

```diff
@@ -281,10 +281,12 @@ func SigningInfoCallback(k *Keeper, ctx sdk.Context, args []byte, query icqtypes
 			return fmt.Errorf("can not get validator address from consensus address: %s", valSigningInfo.Address)
 		}
 
+		fmt.Println("founded")
```

Comment:
I think this is used for debugging right @phamminh0811

Reply:
Yes